### PR TITLE
Add some URLs from Turkish court order

### PR DESCRIPTION
**Pull request by muellermartin**
Today a court order from the Ankara 3rd Criminal Judgeship of Peace
(order no. 2019/5538) was leaked which lists 136 URLs that are ordered
to be blocked in Turkey.

Sources:
- https://bianet.org/system/uploads/1/files/attachments/000/002/687/original/Ankara_3rd_Criminal_Judgeship_of_Peace_2019-5538_Misc..pdf?1565091519
- https://bianet.org/bianet/ifade-ozgurlugu/211382-bianet-in-tum-icerigine-engelleme-karari
- https://umutgazetesi15.org/arsivler/18356
- https://www.zeit.de/politik/ausland/2019-08/internetzensur-tuerkei-oppositionelle-webseiten-sperrung

Only non-duplicate domains and working URLs were added.

### CODE REVIEW

File: lists/tr.csv [hunk 1]
```diff
@@ -178,3 +178,20 @@ https://www.bbc.com/turkce,NEWS,News Media,2018-09-13,OONI,
...
 https://tr.sputniknews.com/,NEWS,News Media,2018-09-13,OONI,
 https://www.amerikaninsesi.com/,NEWS,News Media,2018-09-13,OONI,
+http://etha10.com/,NEWS,News Media,2019-08-06,,Ankara 3rd Criminal Judgeship of Peace court order 2019/5538
+https://ozgurgelecek17.net/,NEWS,News Media,2019-08-06,,Ankara 3rd Criminal Judgeship of Peace court order 2019/5538
+http://osp.org.tr/,POLR,Political Criticism,2019-08-06,,Ankara 3rd Criminal Judgeship of Peace court order 2019/5538
+http://geziyisavunuyoruz.org/,POLR,Political Criticism,2019-08-06,,Ankara 3rd Criminal Judgeship of Peace court order 2019/5538
+https://www.gazetefersude.com/,NEWS,News Media,2019-08-06,,Ankara 3rd Criminal Judgeship of Peace court order 2019/5538
+https://yenidemokratgenclik1.com/,POLR,Political Criticism,2019-08-06,,Ankara 3rd Criminal Judgeship of Peace court order 2019/5538
+http://umutgazetesi14.org/,NEWS,News Media,2019-08-06,,Ankara 3rd Criminal Judgeship of Peace court order 2019/5538
+http://kizilbayrak44.net/,NEWS,News Media,2019-08-06,,Ankara 3rd Criminal Judgeship of Peace court order 2019/5538
```
Comment (hellais) [inline lists/tr.csv]: This site redirects to https when visiting it. We should be sure the HTTPS version of this (and other sites included in this PR that support HTTPS) are used for the list.

Comment (muellermartin) [reply lists/tr.csv]: I simpy copied the URLs from the court order ;) I will check all URLs and change them if they support HTTPS.

**Comment by muellermartin** [comment]
I've tested all URLs if they support HTTPS and changed the scheme if necessary. One site (https://antakyasokak.e-monsite.com/) redirects from HTTPS to HTTP, but maybe that changes in the future, so I've used the HTTPS scheme.

**Comment by hellais** [comment]
Excellent. Thanks a bunch!

Looks good to merge!

**Pull request merged**